dimod/constrained.py

```diff
@@ -287,6 +293,12 @@ def add_constraint_from_model(self,
         Returns:
             The label of the added constraint.
 
+        Examples:
+            >>> from dimod import ConstrainedQuadraticModel, Binary
+            >>> cqm = ConstrainedQuadraticModel()
+            >>> x = Binary('x')
+            >>> cqm.add_constraint_from_model(x, '>=', 0, "Min x")   # doctest: +IGNORE_RESULT
```

Comment:
Why ignore the result? I think a key point is that it returns the constraint label for future reference. If you don't want an output, I would do
```
>>> label = cqm.add_constraint_from_model(x, '>=', 0, label="Min x")
```
to make that clear.

Applies in a bunch of places.

Reply:
Would it not make more sense to do `label = ...` only if one is not setting the label oneself? Otherwise one gets back exactly the string one set

Reply:
Sure, though this time as a variable. I still think it's an important point to make. In any case I wouldn't ignore the output.

Reply:
Done